### PR TITLE
Allow for XHR to be converted into a plain object

### PR DESCRIPTION
**Pull request by ocke**
Allow for XHR to be converted into a plain object with the following properties:

```
readyState
responseText
status
statusText
responseJSON
```

### CODE REVIEW

File: delete_non_clonable.ts [hunk 1]
```diff
@@ -77,6 +77,20 @@ export default function deleteNonClonable(obj: any, depth: number = 0) {
       });
       return newSet;
     }
+    case 'XMLHttpRequest':
+      const newXHR = {
+        readyState: obj.readyState,
+        responseText: obj.responseText,
+        status: obj.status,
+        statusText: obj.statusText,
+        responseJSON: undefined
```
Comment (sarmadsangi) [inline delete_non_clonable.ts]: `responseJSON: undefined `

^ Does this mimic current behavior?

**Comment by Mehul-Porwal** [comment]
Too late here, but why do we have both `package-lock.json` and `yarn.lock` files in this repo?

**Comment by sarmadsangi** [comment]
We moved to npm from yarn. I think olaf just forgot to remove this.

**Comment by Mehul-Porwal** [comment]
Okay, so the `yarn.lock` file is deleted, I just got confused as it was the first PR I did not expect anything to be deleted.

All good 👍

**Pull request merged**